src/client/components/Sidebar/User.less

```diff
@@ -28,9 +28,13 @@
 }
 
 .User__follow {
-  display: flex;
+  width: 40%;
```

Comment:
We shouldn't force a 40% width on a button, it should be dynamic of the container and contents.